Removed note "can be installed on windows only"

lohithgn opened the pull request:
Since Core Tools are now XPlat - removed note which says "it can be installed on windows only"

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Waiting for Larry's approval.

Blackmist commented:
Thanks Dia, looks good - merge.

lohithgn committed:
Removed note

Since Core Tools are now XPlat - removed note which says "it can be installed on windows only"
